crawler/mobile.js

```diff
@@ -148,7 +148,7 @@ mobile.crawl = function(character, full) {
 
     // Number of Tweets after which retry with another "until:(lastDay)" query
     // when the history ends. For some popular characters the history ends early.
-    const retryThreshold = 5000;
+    const retryThreshold = 3000;
```

Comment:
before the questions pops up: calculated this value from crawling Jon Snow last night